tailwind.config.js

```diff
@@ -20,14 +20,21 @@ module.exports = {
       UMOgreen: '#3baa34',
       UMOblue: '#2370b8',
       UMOpurple: '#662684',
+      white: '#ffffff',
   	},
   	textColor: {
        'primary': '#FFFFFF',
        'secondary': '#00B1E9',
        'danger': '#e3342f',
        'question':'#232323',
+       'UMOgreen': '#3baa34',
```

Comment:
perhaps extract this to an override css file? so we can change it for each project

Reply:
using semantic names is better in that case, so instead of 'green' use something like 'primary color'